python/rpdk/java/codegen.py

```diff
@@ -571,3 +572,14 @@ def generate_image_build_config(self, project):
             "dockerfile_path": dockerfile_path,
             "project_path": str(project_path),
         }
+
+    @staticmethod
+    def _get_plugin_information():
+        return {
+            "plugin-version": __version__,
```

Comment:
This is collecting the plugin tool version.  We should also collect the plugin runtime version as well.  

We did similar plugin version check for 1.0 to 2.0 upgrade, there is utils method to retrieve the version. Although customer may not use pom.xml in the package, but it should be a best effort to collect the runtime version.

I think the metadata should like: 

```
{
"plugin-name": "java", 
"plugin-tool-version": "2.0.1",
"plugin-version": "2.0.3"

}
```